resources/views/speakers/index.blade.php

```diff
@@ -0,0 +1,32 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        <div>
+            <h1>Speakers</h1>
+            @auth
+                <a href="{{ route('speakers.create') }}">Add new speakers</a>
+            @endauth
+            @if ($speakers->count())
+                @foreach ($speakers as $speaker)
+                    <div>
+                        {{ $speaker->name }}
+                        {{ $speaker->description }}
+                        {{ $speaker->avatar }}
+                        {{ $speaker->linkedinUrl }}
+                        {{ $speaker->githubUrl }}
+
+                        <a href="{{ route('speakers.edit', $speaker) }}">Edit</a>
+                        <form action="{{ route('speakers.destroy', $speaker) }}" method="post">
```

Comment:
Any confirmation for that action? Smth like [that](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm).